source/deployment/ldap-group-sync.rst

```diff
@@ -0,0 +1,146 @@
+To add the default Team or Channels that you want the group members to default in, select either “Add Team” or “Add Channel” from the blue “Add Team or Channel” button.  You may select multiple teams and channels.  
+
+.. image:: ../images/Add_Team_Or_Channel.png
+
+Channels are nested below the Team they belong to in the team and channel list.  
+
+Teams that are open for anyone to join are indicated by:
+ 
+.. image:: ../images/open_team.png  
+   
+Teams that are not open for anyone to join are indicated by:
+ 
+.. image:: ../images/private_team.png 
+ 
+Public channels are indicated by: 
+ 
+.. image:: ../images/public_channel.png
+
+Private channels are indicated by:
+ 
+.. image:: ../images/private_channel.png  
+
+Teams added include default channels set in the `ExperimentalDefaultChannels config setting <https://docs.mattermost.com/administration/config-settings.html?highlight=configuration%20settings#default-channels-experimental>`_ (Town Square and Off-Topic). Adding a channel without setting the team adds the implied team to the listing below, but not to the group specifically.  Teams are listed in parentheses after the channel name in the channel selector.
+
+Teams and channels membership synchronization
+----------------------------------------------
+
+Default teams and channels will be added to the group member's user interface when they login for the first time after the group has been configured with the specified teams and channels. It may take a few seconds to load all team and channel memberships for a user depending on the number of teams and channels the group is defaulted to. In our testing, it took 6s for an organization with 200,000 users and 30,000 linked groups. This process is independent of the AD/LDAP synchronization process to support future functionality of groups.
+
+.. note::
+Users are not removed from the team or channel on subsequent synchronizations of the AD/LDAP groups.  Users will need to be manually removed from the team or channel per the existing functionality.  They will not be re-added if they were manually removed.
+
+.. image:: ../images/Team_Channel_Membership_Sync.png
+
+Remove configured teams and channels from a group
+-------------------------------------------------
+To remove a team or channel configured for a group, click “Remove” to the right of the team or channel name. 
+
+View users belonging to the group
+---------------------------------
+
+Users who have logged in and accessed Mattermost will be visible in the members list on the group object. Members are read-only at this time and new members can be added through management in your AD/LDAP system. 
+
+.. image:: ../images/Group_Members.png
+
+Users can be removed from the group on subsequent synchronizations. However, they will not be removed from teams and channels. If a user is removed from a AD/LDAP group and then later re-added, they will be defaulted again into the teams and channels configured in the group. 
```

Comment:
Can we switch all "a AD/LDAP" to "an AD/LDAP"?

Reply:
I believe I have them all updated now.